internal/xds/translator/ratelimit.go

```diff
@@ -429,7 +430,10 @@ func (t *Translator) createRateLimitServiceCluster(tCtx *types.ResourceVersionTa
 	clusterName := getRateLimitServiceClusterName()
 	// Create cluster if it does not exist
 	host, port := t.getRateLimitServiceGrpcHostPort()
-	endpoints := []*ir.DestinationEndpoint{ir.NewDestEndpoint(host, uint32(port))}
+	ds := &ir.DestinationSetting{
+		Weight:    ptr.To(uint32(1)),
+		Endpoints: []*ir.DestinationEndpoint{ir.NewDestEndpoint(host, uint32(port))},
+	}
```

Comment:
this can be simplified to a function

Reply:
can this refactor be done in a follow up ?
this PR is changing 255 files most of which are YAML, and will affect other PRs